bucket/windscribevpn-np.json

```diff
@@ -0,0 +1,41 @@
+{
+    "##": "This package is non-portable because the app requires installing drivers/services to work properly.",
+    "version": "2.4.11",
+    "description": "Open Source VPN with AdBlocking Proxy.",
+    "homepage": "https://windscribe.com/",
+    "license": "GPL-2.0",
+    "notes": [
+        "If windscribevpn-np is not installed properly, please run:",
+        "scoop uninstall windscribevpn-np"
+    ],
+    "url": "https://github.com/Windscribe/Desktop-App/releases/download/v2.4.11/Windscribe_2.4.11.exe#/vpnsetup.exe",
+    "hash": "5328DBC9EDDDB1AF7231354C16AC95FA2DE0FC66CFC4F2B3CD57057D42DA0333",
+    "installer": {
+        "script": [
+            "if (!(is_admin)) {error \"$app requires admin rights to $cmd\"; break}",
+            "Start-Process \"$dir\\vpnsetup.exe\" -Wait -ArgumentList \"-silent\", \"-dir $dir\\app\"| Out-Null",

```

Comment:
`Start-Process` does not have a parameter called `-dir`.
(You can try to run the command in *Powershell* to find out)

If you are putting `-dir` as an argument for `vpnsetup.exe`, then use:
`-ArgumentList @('-silent', '-dir', \"$dir\\app\")`

Reply:
I'll try this. Thanks @issaclin32